national-resources/norway.md

```diff
@@ -0,0 +1,176 @@
+
+The FHI advises health authorities on public health issues, including disease prevention, outbreak control, and preparedness for national health emergencies. The institute works with the Directorate of Health to develop vaccination programs, guide infection control measures, and support healthcare providers and related institutions. The [Division of Infection Control](https://www.fhi.no/en/about/this-is-the-norwegian-institute-of-public-health/divisions/infection-control-environmental-health/) within FHI supports infectious disease research, monitors infectious disease outbreaks, and analyses data related to these outbreaks. During the recent pandemic, FHI published a [handbook for medical researchers](https://www.fhi.no/nettpub/overvaking-vurdering-og-handtering-av-covid-19-epidemien-i-kommunen/) to understand the handling of SARS-CoV-2 variants data.
+
+Several governmental organisations collaborate with FHI to monitor pathogenic data and support surveillance initiatives:
+* [The Norwegian Environment Agency](https://www.environmentagency.no) oversees detection of pathogens in wastewater and tracks occurrences of antimicrobial resistance (AMR) in natural areas and wild species. The agency is known as [Miljødirektoratet](https://www.miljodirektoratet.no) in Norwegian and has a Norwegian homepage too.
+* [The Norwegian Food Safety Authority](https://www.mattilsynet.no/language/english/) (Mattilsynet) plays a key role in managing and preventing infectious disease outbreaks that occur within the country’s food chain. The authority guides and drafts legislations related to food and drinking water, oversees the production and distribution processes of all food in Norway, and investigates and tracks pathogens found in drinking water and domestic animals. On behalf of the Food Safety Authority, the national reference laboratories collect data on isolated infectious agents from disease outbreaks related to the food chain and submit it to surveillance databases.
+* [The Norwegian Medicines Agency](https://legemiddelverket.no/English) (Legemiddelverket) regulates and supervises medicine, medical devices, and clinical trials in Norway.
+In addition to the national-level health authorities and agencies, there are regional/municipality-level efforts taking place across hospitals in Norway to combat infectious diseases:
+* The [Norwegian National Advisory Unit on Tropical Infectious Diseases](https://helse-bergen.no/nasjonal-kompetansetjeneste-for-tropiske-infeksjonssykdommer/norwegian-national-advisory-unit-on-tropical-infectious-diseases) contributes to tropical infectious disease research, standards, and training. They are responsible for reporting on and mapping imported tropical infectious diseases as part of a larger European network.
```

Comment:
This link is broken, unsure if the page has moved recently?

Reply:
Fixed it. Thanks for pointing it out!